gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/GapicClass.java

```diff
@@ -26,7 +27,8 @@ public enum Kind {
     MAIN,
     STUB,
     TEST,
-    PROTO
+    PROTO,
+    NON_GENERATED
```

Comment:
This is fine to me. Can you please add more comments to explain what `NON_GENERATED` means?

Reply:
Sure, will add.